src/components/programs/bitbyte/faq.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+
+const FAQ = () => {
+  return (
+    <div>
+      <p className="text-acm-gray-500 m-[20] text-center text-5xl font-bold">
```

Comment:
```suggestion
      <p className="text-acm-gray-500 m-[20] text-center text-5xl font-bold">
```
pick a tailwind class don't just do m-[20]